src/components/App/App.css

```diff
@@ -30,18 +30,8 @@ button {
   border: none;
   height: 70%;
   width: 80%;
-}
-
-.searchBar {
-  position: fixed;
-  background-color:gray;
+  background-color: black;
```

Comment:
Don't forget to make sure you've used some consistent method for determining the order of your CSS properties inside each rule.